features/account/customer_account/viewing_orders_history/changing_payment_method.feature

```diff
@@ -19,3 +20,8 @@ Feature: Changing a payment method of a placed order
     Scenario: Changing a payment method of an order
         When I browse my orders
         Then I should be able to change payment method for this order
+
+    @api
+    Scenario: Changing a payment method of an order
```

Comment:
IMHO you should merge these two scenarios

Reply:
also, those two scenarios should be refactored, we need to check "changing payment method" and not a possibility to change the payment method.
sth like:
```
When I browse my orders
And i change my payment method to "Bank Transfer"
Then I should have "Bank Transfer" payment method on my order